src/generic.cpp

```diff
@@ -1643,6 +1643,23 @@ unsigned int OBVibrationData::GetNumberOfFrequencies() const
   return this->_vFrequencies.size();
 }
 
+bool OBPcharges::FindPcharge(int    atomIndex,
+                             double *q)
```

Comment:
This is hugely slow. It's completely un-needed if the charges are kept as vector<double> in the OBPCharges class.